app/controllers/certificates_controller.rb

```diff
@@ -1,30 +1,27 @@
 class CertificatesController < ApplicationController
-  
-  def index
-    @uploads = Certificate.all
-  end
-
-  def upload
+  def new
     @upload = UploadCertificateEvent.new
   end
 
   def create
     @upload = UploadCertificateEvent.create(upload_params)
+
     if @upload.valid?
-      redirect_to certificates_path
+      redirect_to component_path @upload.component_id
```

Comment:
can we add parenthesis here?
```suggestion
      redirect_to component_path(@upload.component_id)
```

Reply:
👍